examples/wifi-echo/server/esp32/main/ember-print.cpp

```diff
@@ -0,0 +1,68 @@
+#include <stddef.h>
+#include <stdio.h>
+#include <string.h>
+
+#include <support/CHIPLogging.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+bool emberAfPrintReceivedMessages = true;
+
+using namespace chip::Logging;
+void emberAfPrint(int category, const char * format, ...)
```

Comment:
this function probably needs to die, because we don't really have a facility to emit a log entry without a newline.

maybe we can save this off in a global variable, and emit the saved string in PrintLn()

Reply:
This is just to port over the Silabs code as is without making custom modifications. 
So let's keep this for now and we can have a separate PR to kill this if needed once we are done with the overall port.